File: server/ToolBox.Services.SQLMonitor/Domain/Models/SqlServerBadge.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ToolBox.Services.SQLMonitor.Domain.Models
{
    public class SqlServerBadge
    {
        public Guid ServerId { get; set; }
        public string Name { get; set; }
        public string ServerAddress { get; set; }
        public string Description { get; set; }
        public double PageReadsCounts { get; set; }
        public double RequestCount { get; set; }
        public double PageLifetime { get; set; }
        public bool IsAlive = true;

        public int ConnectedUsers = 0;
    }
}

File: server/ToolBox.Client/ClientApp/src/app/_models/sql-server-badge.ts
export class SqlServerBadge{
     serverId:string;
     name:string;
     serverAddress:string; 
     description :string;
     pageReadsCount :number = 0;
     requestCount :number = 0;
     pageLifetime :number = 0;
     connectedUsers:number = 0;
     isAlive:boolean = true;

     /**
      *
      */
     constructor() {
     this.name ='';
     this.serverAddress =''; 
     this.description='';
     this.pageReadsCount = 0;
     this.requestCount  = 0;
     this.pageLifetime = 0;
     this.connectedUsers = 0;
     this.isAlive = true;
     }
}